lib/ansible/modules/remote_management/oneview/oneview_datacenter_facts.py

```diff
@@ -39,13 +39,19 @@
 EXAMPLES = '''
 - name: Gather facts about all Data Centers
   oneview_datacenter_facts:
-    config: "/etc/oneview/oneview_config.json"
+    hostname: 172.16.101.48
+    username: administrator
+    password: my_password
+    api_version: 500
   delegate_to: localhost
```

Comment:
The no_log should be set to true here, as it is not a best practice to be printing the user password on every execution